credoai/governance/governance.py

```diff
@@ -140,7 +142,7 @@ def register(
             self.set_evidences([])
 
     def __parse_json_api(self, json_str):
-        return deserialize(json.loads(json_str))
+        return json.loads(json_str)
```

Comment:
@aproxacs, I was getting an error with deserialization when using the file taken straight from the platform.

<img width="903" alt="image" src="https://user-images.githubusercontent.com/85892367/191421406-0c5a5ddb-e3a4-48ce-8977-e543c43cc274.png">

I think it's because the platform already deserializes before downloading. Is that true? If so, this fixes the bug.

Reply:
yeah, right, I am fixing this. UI app should return original JSON API format.

Reply:
Ok, I'll change this code back then